reports-backend/src/main/java/org/jboss/da/reports/backend/impl/DependencyTreeGeneratorImpl.java

```diff
@@ -40,13 +40,9 @@ public GAVDependencyTree getDependencyTree(SCMLocator scml) throws ScmException,
     }
 
     @Override
-    public Optional<GAVDependencyTree> getDependencyTree(GAV gav) throws CommunicationException {
-        try {
-            return Optional.of(aproxConnector.getDependencyTreeOfGAV(gav));
-        } catch (FindGAVDependencyException e) {
-            // TODO: better handle this later in DA-170
-            return Optional.empty();
-        }
+    public Optional<GAVDependencyTree> getDependencyTree(GAV gav) throws CommunicationException,
+            FindGAVDependencyException {
+        return Optional.of(aproxConnector.getDependencyTreeOfGAV(gav));
```

Comment:
When it never returns empty optional, I think we can change the return type to just ``GAVDependencyTree``